na-toolbar.html

```diff
@@ -49,8 +49,12 @@
     </style>
     <iron-icon icon="bookmark"></iron-icon>
     <h1>Notes</h1>
-    <iron-icon icon="cloud" hidden$="[[!online]]"></iron-icon>
-    <iron-icon icon="cloud-off" hidden$="[[online]]"></iron-icon>
+    <span hidden$="[[!online]]">
+      <iron-icon icon="cloud"></iron-icon>
+    </span>
+    <span hidden$="[[online]]">
+      <iron-icon icon="cloud-off"></iron-icon>
+    </span>
```

Comment:
Oh, `display` in `:host` beats the `hidden` attribute (which only has UA stylesheet precedence). I think the right way to handle this is to implement `[hidden]` manually: https://github.com/PolymerElements/paper-fab/blob/c4afc403d3bd2caa3caea92629f533ba338ab6f6/paper-fab.html#L89 . (Maybe `!important` isn't necessary in this case?)

I think this works in 1.x because of this: https://github.com/PolymerElements/iron-flex-layout/pull/103/files#diff-ac9bc15ef746292eee06aa75286380c2L51